src/mockNetwork.ts

```diff
@@ -94,4 +95,11 @@ const mockNetwork = (responses: Response[] = [], debug: boolean = false) => {
   fetch.mockImplementation(request => mockFetch(responses, request, debug))
 }
 
+const printMultipleResponsesWarning = (response: Response) => {
+  const errorMessage = `Missing response in the multipleResponses array for path ${response.path} and method ${response.method}.`
```

Comment:
What do you think using the same message structure as we defined in the messages file (`src/assertions/messages.js`) something like:

```suggestion
  const errorMessage = `🌯 Wrapito:  Missing response in the multipleResponses array for path ${response.path} and method ${response.method}.`
```

Also we can move this message to this file or create a new messages file with its own context, what do you thing?

Reply:
Just committed a fix for the message structure. As for where to place the message, `messages.js` doesn't feel like the proper place as it only contains assertion related logs. At the same time, I'm not sure about the need of creating a new `messages.js` file for the only two log messages that we're printing from `mockNetwork` 🤔 If you're certain about the purpose of the new file and where to place it tho, we can definitely try it out!

Reply:
Ok, at the moment isn't a pain, so we can continue like this and in the future, we can speak about it if it makes sense to move it